cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +634,47 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1460788)])
+def test_service_ansible_verbosity(
```

Comment:
@sbulage After talking with Mike, this should be paramertized from `5-0` to test all the available options. Also add the finalizer back in to reset it to the default string as @mshriver suggested below. Sorry to steer your wrong with the logic of setting it back to 0 below.

Reply:
Added Parametrization for all available Verbosity. Resetting back to `0 (Normal)`.